src/components/chatFriends/ChatFriendsList.tsx

```diff
@@ -0,0 +1,55 @@
+import styled from "styled-components";
+import user from "data/user.json";
+import List from "components/layout/List";
+import Search from "components/common/Search";
+import { useState } from "react";
+
+const ChatList = () => {
+  const totalUser = user.slice(1);
+  const [showUser, setShowUser] = useState(totalUser);
+
+  const filterUser = (text: string): void => {
+    if (!text.trim()) {
+      setShowUser(totalUser);
+    } else {
+      const filtered = totalUser.filter((user) =>
+        user.name.toLowerCase().includes(text.trim().toLowerCase())
+      );
+      setShowUser(filtered);
+    }
+  };
```

Comment:
```suggestion
  const filterUser = (text: string): void => {
    const filtered = totalUser.filter((user) =>
      user.name.toLowerCase().includes(text.trim().toLowerCase())
    );
    setShowUser(filtered);
  };
```

우선 trim을 통해 공백을 처리하신 점이 인상 깊습니다. 다만, 입력이 없으면 모든 includes 값이 true로 나와서 별도로 전체 보여주기를 처리할 필요가 없더라구요.

Reply:
오 그렇네요! 알려주셔서 감사합니다~!